docs/site/src/components/API/api-ref/refnav.js

```diff
@@ -41,10 +41,9 @@ const RefNav = (props) => {
                 .map((method) => {
                   return (
                     <Link
-                      className={`menu__link font-medium block hover:no-underline text-base ${page === method.name && 'menu__link--active'}`}
+                      className={`menu__link font-medium block  hover:no-underline text-base ${page === method.name && 'menu__link--active'}`}
```

Comment:
Is there a reason for the extra blank space?

Reply:
It's a typo, my bad.